structural-variantcalling.wdl

```diff
@@ -0,0 +1,164 @@
+                newSampleName = sample + "." + pair.right 
+       }
+   }
+   
+   call survivor.Merge as survivor {
+       input:
+            dockerImage = dockerImages["survivor"],
+            filePaths = renameSample.renamedVcf,
+            outputPath = outputDir + '/structural-variants/survivor/' + sample + '.merged.vcf'
+   }
+   
+   output {
+        File cleverPredictions = clever.predictions
+        File cleverVcf = mateclever.matecleverVcf
+        File mantaVcf = manta.mantaVCF
+        File dellyBcf = delly.dellyBcf
+        File dellyVcf = delly2vcf.outputVcf
+        File survivorVcf = survivor.mergedVcf
+        Array[File] renamedVcfs = renameSample.renamedVcf 
+   }
+
+   parameter_meta {
+        outputDir: {description: "The directory the output should be written to.", category: "common"}
+        referenceFasta: { description: "The reference fasta file", category: "required" }
+        referenceFastaFai: { description: "Fasta index (.fai) file of the reference", category: "required" }
+        referenceFastaDict: { description: "Sequence dictionary (.dict) file of the reference", category: "required" }
+        bamFile: {description: "sorted BAM file", category: "required"}
+        bamIndex: {description: "BAM index(.bai) file", category: "required"}
+        bwaIndex: {description: "Struct containing the BWA reference files", category: "required"}
+        sample: {description: "The name of the sample", category: "required"}
+        memory: {description: "The memory required to run the programs", category: "common"}
+        memoryGb: {description: "The memory required to run the manta", category: "common"}
+        cores: {description: "The the number of cores required to run a program", category: "common"}
+        threads: {description: "The the number of threads required to run a program", category: "common"}
+        javaXmx: {description: "The max. memory allocated for JAVA", category: "common"}
+        dockerImagesFile: {description: "A YAML file describing the docker image used for the tasks. The dockerImages.yml provided with the pipeline is recommended.",
+                           category: "advanced"}
+                           
+        minSize: {description: "The mimimum size of SV to be merged", category: "required"}
+        distanceBySvSize: {description: "A boolean to predict the pairwise distance between the SVs based on their size", category: "required"}
+        strandType: {description: "A boolean to include strand type of an SV to be merged", category: "required"}
+        svType: {description: "A boolean to include the type SV to be merged", category: "required"}
+        suppVecs: {description: "The minimum number of SV callers to support the merging", category: "required"}
+        breakpointDistance: {description: "The distance between pairwise breakpoints between SVs", category: "required"}
```

Comment:
Could you ensure that only the inputs which are actually defined in the workflow are mentioned here. `javaXmx` for example is not a workflow input.

Please also double check the categories.